### PR TITLE
Update How to Install and Use Geospatial R Packages.md

### PR DESCRIPTION
**Pull request by Moohan**
I added some text / code on how to update your/a project's .Rprofile

The guide includes a few bits of code that need to be run in every session to make geospatial packages work. It does at one point suggest adding code to a porject's .Rprofile but it doesn't specify *how* to do this. The other issue is that the code that should be included in an .Rprofile is split accross the guide with the env variables at the top and the DLL loads at the bottom, so it would be easy for someone to miss this and only add part of the required code to an .Rprofile.

I also included some code to 'automatically' add the text to a (personal) .Rprofile, although I'm in two minds as to whether this lowers the bar 'too much'.

### CODE REVIEW

**Review by rmccreath — APPROVED** [review]

**Comment by Moohan** [comment]
Am I ok to merge this?

I checked the contribution guidelines and there's not really a recommended flow for PRs. I would suggest:

- Require a 2 approved reviews (add branch protection)
- The second approver should merge (or squash and merge?) the PR, or should tag a third reviewer in a comment (if required), the third reviewer should merge on approval or ...
- Some people (DS team?) should be able to override these requirements to be able to push through very simple PRs (typo fixing etc.) - Also done with branch protection.

**Comment by terrymclaughlin** [comment]
@Moohan 

There's branch protection enabled on the main branch, requiring a pull requested and 1 approval before merging.  I have increased this to 2 approvals.

**Review by terrymclaughlin — COMMENTED** [review]
@Moohan 

I think this all looks fine, but I wanted to check one thing before I approve and merge: if somebody includes the specified R code in their personal .Rprofile, and they subsequently uninstall geospatial packages, does that lock up their R sessions?  Equally, if they include the R code in the project's .Rprofile, does that lock out anybody from that project who does not already have geospatial packages installed?

**Comment by Moohan** [comment]
@terrymclaughlin I just tested both of these cases (I switched to 3.6.1 where I have no packages installed which I assume is sufficient?), and it works fine. The only lines I was worried about were the `dyn.load(...)` but it seems as with the other bits that code can all just run but is only relevant when you try to load / install a geospatial package.

File: Posit Infrastructure/How to Install and Use Geospatial R Packages.md
Comment (JFix89) [inline Posit Infrastructure/How to Install and Use Geospatial R Packages.md:218]:
```suggestion
<summary>This code will automatically add the required code (click to expand)</summary>
```

By default, it looks like this section is collapsed when viewing the .md. Maybe just add this to make absolutely clear to users?

Comment (Moohan) [reply Posit Infrastructure/How to Install and Use Geospatial R Packages.md]: Yep it was deliberately collapsed but I agree your comment makes it clearer!

**Review by JFix89 — APPROVED** [review]

**Pull request merged**
